product-scenarios/1-manage-public-partner-private-apis/1.1-expose-service-as-rest-api/1.1.1-create-rest-api-from-scratch/src/test/java/org/wso2/am/scenario/tests/rest/api/creation/RESTApiCreationTestCase.java

```diff
@@ -52,25 +65,22 @@
     private String techOwner = "wso2";
     private String techOwnerMail = "wso2@gmail.com";
     private String endpointType = "secured";
-    private String endpointAuthType = "basicAuth";
+    private String endpointAuthType = "BASIC";
     private String epUsername = "wso2";
     private String epPassword = "wso2123";
-    private String default_version_checked = "default_version";
-    private String responseCache = "enabled";
+    private boolean default_version_checked = true;
```

Comment:
Shall we make this variable **defaultVersionChecked** instead of **default_version_checked**, since it is the most common way to define variables?

Reply:
Fixed with https://github.com/wso2/product-apim/pull/9704